app/views/calls/_new_call.html.erb

```diff
@@ -2,15 +2,19 @@
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-body">
+      <div class="calls-layout">
         <button type="button" class="close close-terms" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title">Call <b><%= p.patient.name %></b> now</h4>
-        <h4><%= p.patient.primary_phone %></h4>
-        <%= button_to "I reached the patient", pregnancy_calls_path(p, call: { status: 'Reached patient'} ), method: :post, class: 'btn btn-primary' %>
-        </br>
-        <%= link_to "I left a voicemail for the patient", pregnancy_calls_path(p, call: { status: "Left voicemail" } ), method: :post, remote: true %>
-        </br>
-        <%= link_to "I couldn't reach the patient", pregnancy_calls_path(p, call: { status: "Couldn't reach patient" } ), method: :post, remote: true %>
-      </div>
+       <h4 class="modal-title calls-request">Call <b><%= p.patient.name %></b> now:</h4>
+        <br>
+        <h4 class="calls-phone"><%= p.patient.primary_phone %></h4>
+        <br>
+        <%= button_to "I reached the patient", pregnancy_calls_path(p, call: { status: 'Reached patient'} ), method: :post, class: 'btn btn-primary', id: 'calls-btn' %>
+       <br><br>
```

Comment:
@dfelsent I wonder if we're leaning a little too hard on `<br>` here -- thoughts on doing this using bootstrap classes rather than line breaks so it's cleaner? (If there's not an obvious path to it let's punt for now.)